src/NodaTime.Demo/YearMonthDemo.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright 2019 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using NUnit.Framework;
+using System;
+
+namespace NodaTime.Demo
+{
+    public class YearMonthDemo
+    {
+        [Test]
```

Comment:
There's no documented constructor for this to attach to. I'd suggest removing it to avoid confusion.

Reply:
Got it.